homeassistant/components/lovelace/__init__.py

```diff
@@ -26,6 +26,7 @@
     }),
 }, extra=vol.ALLOW_EXTRA)
 
+EVENT_LOVELACE_UPDATED = 'lovelace_updated'
```

Comment:
Make sure you whitelist this event in `websocket_api/permissions.py` or else only admin users can listen to it.